### PR TITLE
BUG: fix explore() for older folium + test with folium 0.11

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Potentially fixes https://github.com/geopandas/geopandas/issues/2187. Just a quick PR to see what CI does with this.

### CODE REVIEW

File: geopandas/explore.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def my_colormap(value):  # scalar value defined in 'column'
     if gdf.crs is None:
         kwargs["crs"] = "Simple"
         tiles = None
-    elif not gdf.crs.equals(4326):
+    elif not gdf.crs == "EPSG:4326":
```
Comment (jorisvandenbossche) [inline geopandas/explore.py:285]: The `equals` method was only added in pyproj 2.5 (we currently require 2.2, although we should probably bump that)

**Comment by martinfleis** [comment]
Okay, not passing the marker to GeoJson for older versions is fine. It is just hard to do that at the moment as older folium has broken `__version__`. It returns `'unknown'`. That is why CI errors. So we'll likely need some try-except handling here, because this has been fixed only recently (xref https://github.com/python-visualization/folium/issues/1513).

**Comment by martinfleis** [comment]
I will close this as folium 0.12 has been released 2 years ago so fixes for older versions are no longer within our support period. Also, the default version on Google Colab is now 0.12.1.post1 so these issues should no longer pop up that often.

**Comment by jorisvandenbossche** [comment]
Sounds good!

**Pull request closed without merging**